src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/TensorsInfo.cs

```diff
@@ -226,17 +280,26 @@ public int[] GetDimension(int idx)
         /// <since_tizen> 6 </since_tizen>
         public TensorsData GetTensorsData()
         {
-            IntPtr tensorsData_h;
+            IntPtr tensorsData_h = IntPtr.Zero;
             TensorsData retTensorData;
             NNStreamerError ret = NNStreamerError.None;
 
+            NNStreamer.CheckNNStreamerSupport();
+
             if (_handle == IntPtr.Zero)
             {
                 Log.Info(NNStreamer.TAG, "_handle is IntPtr.Zero\n" + "  GetTensorsInfoHandle() is called");
                 GetTensorsInfoHandle();
             }
 
-            ret = Interop.Util.CreateTensorsData(_handle, out tensorsData_h);
+            try
+            {
+                ret = Interop.Util.CreateTensorsData(_handle, out tensorsData_h);
+            }
+            catch (DllNotFoundException e)
```

Comment:
DllNotFound is already checked in CheckNNStreamerSupport()

Reply:
ACK